draft-ietf-jsonpath-base.md

```diff
@@ -1063,27 +1063,27 @@ descendant-selector = ".." ( dot-member-name      /  ; ..<name>
                            )
 ~~~~
 
-Note that there is no "bald" descendant selector `..`.
+Note that there is no "bald" descendant selector (`..` on its own).
```

Comment:
I'm not sure we need to introduce that term (even though, informally, I find it amusing).

```suggestion
Note that `..` on its own is not a valid selector.
```

Reply:
Not a good suggestion?

Reply:
Whoops! Yes, it was a good suggestion, thanks. Since it's small, I'll piggy-back it on another PR.

Reply:
See https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/214/commits/c7c245a95302c5e792d34e0ba4c6c9de10e431b2